src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs

```diff
@@ -520,7 +520,7 @@ private void UpdateConnectorDataToolTip()
                 var portValue = model.Start.Owner.GetValue(model.Start.Index, workspaceViewModel.DynamoViewModel.EngineController);
                 if (portValue is null)
                 {
-                    ConnectorDataTooltip = "N/A";
```

Comment:
This can also go back to "N/A" but seems like we should just not show a tooltip if there is nothing to show